vms/shift/tests/test_shiftHours.py

```diff
@@ -126,8 +127,12 @@ def test_edit_hours(self):
         completed_shifts_page.go_to_completed_shifts()
 
         completed_shifts_page.edit_hours('10:00', '13:00')
-        self.assertEqual(completed_shifts_page.get_shift_start_time(), '10 a.m.')
```

Comment:
These checks can still be included right?  to be sure that the hours are updated in UI too.

Reply:
line 129 should automatically send a mail right?